tests/cephfs/snapshot_clone/clone_attributes.py

```diff
@@ -1,7 +1,10 @@
+import json
```

Comment:
@AmarnatReddy you mentioned about open BZ https://bugzilla.redhat.com/show_bug.cgi?id=2332614 for this workflow, do you want to include BZ as comment in relevant test suite where this automation is included until BZ is fixed?

Reply:
BZ is RFE Suma.